File: documentation/user/en/query/filtering/examples/price/price-between.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		HierarchyWithin(
        			"categories",
        			AttributeEquals("code", "e-readers")
        		),
        		PriceInPriceLists("basic"),
        		PriceInCurrency(new Currency("EUR")),
        		UserFilter(
        			PriceBetween(150.0m, 170.5m)
        		)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code"),
        			PriceContentRespectingFilter()
        		)
        	)
        )
	)
);

File: documentation/user/en/query/filtering/examples/price/price-between.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					hierarchyWithin(
						"categories",
						attributeEquals("code", "e-readers")
					),
					priceInPriceLists("basic"),
					priceInCurrency(Currency.getInstance("EUR")),
					userFilter(
						priceBetween(new BigDecimal("150.0"), new BigDecimal("170.5"))
					)
				),
				require(
					entityFetch(
						attributeContent("code"),
						priceContentRespectingFilter()
					)
				)
			)
		);
	}
);